os/board/rtl8730e/src/Makefile

```diff
@@ -0,0 +1,234 @@
+############################################################################
+
+-include $(TOPDIR)/Make.defs
+
+DEPPATH = --dep-path .
+VPATH = .
+ASRCS =
+
+DEPPATH += --dep-path $(TOPDIR)/board/common
+VPATH += :$(TOPDIR)/board/common
+
+ifeq ($(CONFIG_FLASH_PARTITION),y)
+CSRCS += partitions.c
+endif
+
+ifeq ($(CONFIG_BOARD_CRASHDUMP),y)
+CSRCS += crashdump.c
+endif
+
+# boardctl support
+ifeq ($(CONFIG_LIB_BOARDCTL),y)
+DEPPATH += --dep-path $(TOPDIR)/arch
+VPATH += :$(TOPDIR)/arch
+CSRCS += boardctl.c
+endif
+
+include Make.defs
+include component/soc/amebad2/app/monitor/ram/Make.defs
+include component/soc/amebad2/app/monitor/rom/Make.defs
+include component/soc/amebad2/fwlib/ram_common/Make.defs
+include component/soc/amebad2/fwlib/ap_peripheral/Make.defs
+include component/soc/amebad2/fwlib/ap_core/Make.defs
+include component/soc/amebad2/fwlib/rom_common/Make.defs
+include component/soc/amebad2/swlib/Make.defs
+include component/ssl/mbedtls-2.16.9/library/Make.defs
+# Need to check this path
+include component/usrcfg/rtl8730e/Make.defs
+include component/lwip/api/Make.defs
+include component/wifi/Make.defs
+include component/wifi/inic/Make.defs
+include component/wifi/driver/core/Make.defs
+#include component/ssl/mbedtls_ram_map/rom/Make.defs
+include component/mbed/targets/hal/rtl8730e/Make.defs
+include component/os_dep/Make.defs
+include component/os/tizenrt/Make.defs
+
+ifeq ($(CONFIG_AMEBASMART_WIFI),y)
+include component/wifi/api/Make.defs
+include component/wifi/promisc/Make.defs
+include component/wifi/wpa_supplicant/wpa_supplicant/Make.defs
+include component/soc/amebad2/misc/Make.defs
+endif
+
+ifeq ($(CONFIG_AMEBASMART_BLE),y)
+include component/bluetooth/api/Make.defs
+include component/bluetooth/api/ipc/Make.defs
+include component/bluetooth/os/tizenrt/Make.defs
+include component/bluetooth/example/utils/Make.defs
+ifeq ($(CONFIG_AMEBASMART_BLE_PERIPHERAL),y)
+include component/bluetooth/example/ble_peripheral/Make.defs
+endif
+ifeq ($(CONFIG_AMEBASMART_BLE_CENTRAL),y)
+include component/bluetooth/example/ble_central/Make.defs
+include component/bluetooth/profile/client/Make.defs
+endif
+ifeq ($(CONFIG_AMEBASMART_BLE_SCATTERNET),y)
+include component/bluetooth/example/ble_scatternet/Make.defs
+endif
+endif
+
+ARCH_SRCDIR = 	$(TOPDIR)/arch/$(CONFIG_ARCH)/src
+BOARD_SRCDIR = 	$(TOPDIR)/board
+PROJ_DIR = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/project/realtek_amebaD2_va0_example
+COMPONENT_DIR = $(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component
+DIR_WIFI = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/wifi
+DIR_OSDEP = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/os_dep
+DIR_OS = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/os
+DIR_MBED = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/mbed/targets/hal/rtl8730e
+ifeq ($(CONFIG_AMEBASMART_BLE),y)
+DIR_BT = 	$(BOARD_SRCDIR)/$(CONFIG_ARCH_BOARD)/src/component/bluetooth
+endif
+
+TARGETDIR = 	$(COMPONENT_DIR)/soc/amebad2
+
+CFLAGS += -I$(DIR)/../include
```

Comment:
What is the `DIR` definition?

Reply:
Dear @sunghan-chang ,
Yes, here is a mistake, we mean TizenRT\os\board\rtl8730e\include
So I think I can change to:
CFLAGS += -I$(BOARD_SRCDIR)/rtl8730e/include

Reply:
Could you modify this to merge?

Reply:
Yes, I modified this to make it merge.